sdk/communication/communication-identity/test/public/utils/recordedClient.ts

```diff
@@ -34,6 +34,7 @@ export interface RecordedClient<T> {
 
 const replaceableVariables: { [k: string]: string } = {
   COMMUNICATION_CONNECTION_STRING: "endpoint=https://endpoint/;accesskey=banana",
```

Comment:
leaving COMMUNICATION_CONNECTION_STRING since it is used in samples, and samples are run as part of the livetests - it has to be linked to the dynamic resource in order to work.

Reply:
hmm, but this is in the recorder, and that shouldn't have any effect on the samples